php/libraries/NDB_Menu_Filter.class.inc

```
@@ -855,10 +847,6 @@ class NDB_Menu_Filter extends NDB_Menu
         }
         $query .= $this->order_by;
 
-        if (!empty($this->limit)) {
```

I couldn't find any modules making use of this either.

---

(if there were, it should cause phan to fail since this PR is enabling phan detecting undefined property accesses..)